public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -178,14 +197,19 @@ export class PanelInspector extends PureComponent<Props, State> {
       };
     });
 
-    const processed = applyFieldOverrides({
-      data,
-      theme: config.theme,
-      fieldOptions: { defaults: {}, overrides: [] },
-      replaceVariables: (value: string) => {
-        return value;
-      },
-    });
+    const processed = transformDataFrame(
+      [{ id: transformation, options: [] }],
```

Comment:
The seriesToColumn transformations takes { byField: string } as options, you should specify 'time' for it here :)

Reply:
https://github.com/grafana/grafana/blob/5307cfeabd12dd2bc6ab53a507c72a9962cb6737/packages/grafana-data/src/transformations/transformers/seriesToColumns.test.ts#L30

Reply:
Tried this but no success.